leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/serialization/LwM2mPeerSerDes.java

```diff
@@ -28,8 +28,8 @@ public class LwM2mPeerSerDes {
 
     // LwM2mPeer keys
     protected static final String KEY_IDENTITY = "identity";
-    // TODO should we add a type field here ?
-
+    protected static final String KEY_LWM2MPEERTYPE = "type";
+    protected static final String DEFAULT_LWM2MPEERTYPE = "ippeer";
```

Comment:
I'm not sure about the constant name ? why not  : 
 - `KEY_LWM2MPEER_TYPE`
 - `LWM2MPEER_TYPE_IP` ? 
 
(Same logic you used at LwM2mIdentitySerDes)

Maybe the value could also be just `ip` ?

Reply:
> I'm not sure about the constant name ? why not :
> 
>     * `KEY_LWM2MPEER_TYPE`
> 
>     * `LWM2MPEER_TYPE_IP` ?
> 
> 
> (Same logic you used at LwM2mIdentitySerDes)

OK maybe it's better to unify it. 
 
> Maybe the value could also be just `ip` ?

"ip" also  sounds better than "ippeer"